tests/cypress/support/commands.js

```diff
@@ -0,0 +1,68 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add('login', (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add("clickPublish", () => {
+	cy.get(".editor-post-publish-panel__toggle").click();
+});
+
+Cypress.Commands.add("editBlockAsHTML", (id) => {
+	cy.get(`#${id}`).focus();
+
+	// Open HTML editor.
+	cy.get(
+		'.block-editor-block-contextual-toolbar[aria-label="Block tools"]'
+	).within(() => {
+		cy.get(
+			'.components-button[aria-label="Options"], .components-button[aria-label="More options"]'
+		).click();
+	});
+	cy.get(".components-button").contains("Edit as HTML").click();
+});
+
+Cypress.Commands.add("insertInsecureBlock", (after) => {
+	cy.insertBlock("core/image").then((id) => {
+		cy.get(`#${id} .components-form-file-upload input[type=file]`).attachFile(
```

Comment:
We may change `.attachFile()` to `.selectFile()` here and remove the `cypress-file-upload` dependency as `.selectFile()` is available from Cypress 9.3 

More info: https://docs.cypress.io/guides/references/migration-guide#Migrating-from-cypress-file-upload-to-selectFile

Reply:
Thank you for this suggestion!